src/components/List/ListItem/ListItem.scss

```diff
@@ -0,0 +1,27 @@
+.ListItem {
+    &__container {
+        display: flex;
+        align-items: baseline;
+        padding: 0 var(--spacing-3x);
+    }
+
+    &:hover {
+        background-color: var(--color-brand-10);
+    }
+
+    &--isSelected {
+        background-color: var(--color-brand-10);
+    }
+
+    &--isDividedList:not(:last-child) {
+        border-bottom: 1px solid var(--color-neutral);
```

Comment:
Can you make the division border color lighter please? `var(--color-neutral-25)` should do - in general, you can take the tryout I made (65e8f88c0c8489188462b3344d828c3026e7ee5c) as a visual reference if that helps you

Reply:
done